### ISSUE TITLE
[Sourcing] ACC-271.1(Level A) Fdp-table dialog boxes have incorrect initial focus

### ISSUE THREAD

**Issue by devika-devanand** [SAP/fundamental-ngx#11465]
#### Is this a bug, enhancement, or feature request?
Accessibility bug

#### Describe your proposal.
The initial focus on the column settings, sort and filter by dialog boxes are on the 'Cancel' button, which is incorrect.

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
fundamental-ngx version : 0.43.11

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Create a fdp-table with the sort, filter and column settings functionality. 
Open the respective dialog boxes and check for the initial visible focus.
<img width="505" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/71935676/641b9957-9a8b-4a12-9985-4e2cd7a985fd">
<img width="720" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/71935676/b78f08c1-7b69-4908-9a75-356c512279c4">
<img width="424" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/71935676/8d1c3ba2-d016-4acc-9059-b49fe73c5dfb">

#### Please provide relevant source code (if applicable).
Create an fdp-table with the sort, filter and column settings functionality. 
<img width="401" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/71935676/80ad05a1-2e31-4c93-8a4b-db0572087edf">

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Always the initial focus should be present on the first interactive element or the element which has the high importance level.
